publish.js

```diff
@@ -115,21 +118,21 @@ function buildMetadata (spec, auth, registry, manifest, tardata, opts) {
     readme: manifest.readme || ''
   }
 
-  if (opts.access) root.access = opts.access
+  if (access) root.access = access
 
   if (!auth.token) {
     root.maintainers = [{ name: auth.username, email: auth.email }]
     manifest.maintainers = JSON.parse(JSON.stringify(root.maintainers))
   }
 
   root.versions[manifest.version] = manifest
-  const tag = manifest.tag || opts.tag
+  const tag = manifest.tag || _tag
```

Comment:
Wait, what?  You can put a `tag` field in package.json, and it'll use that??  I've been doing `publishConfig: { tag: ... }` forever like an animal.  TIL!

Reply:
That is not even slightly documented in [`npm help package.json`](https://docs.npmjs.com/files/package.json).